### DISCUSSION TITLE
r5 4600u 有能用的补帧吗?

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#55] [Q&A 问答]
这cpu解码4k没问题的
shift+4或者shift+5两种补帧直接cpu100%起飞不敢开了
本来amd自己有补帧但是mpv好像不支持,还有别的办法吗

另外问一下有没有真人电影适合的补帧配置文件啊

**Comment by hooke007** [comment]
二和四都属于低开销了。。。升级配置吧

真人视频太复杂，暂无方案，我都是一号凑活用的

**Reply by hooke007** [reply]
AFM补帧被amd自己淘汰了，RDNA已经不支持了（apu因为挤牙膏的原因还在用GCN

**Reply by jiang-yuan** [reply]
我9750H 用这两个补帧占用率才50%左右啊,看跑分这个cpu和4600u差不了多少,为啥差距那么大呢

4k加补帧5方案,9750H和1660ti也才分别占20%多吧

**Reply by natural-harmonia-gropius** [reply]
我之前用别人的4800h占用也很低，但风扇会起飞。4k补帧5我用5900x也带不动，降到1080p也才能带个180fps左右

**Reply by jiang-yuan** [reply]
> 

我就补到60而已,你5900x不至于带不动吧

**Reply by natural-harmonia-gropius** [reply]
真带不动，感觉svp只用了很小一部分的cpu后端但是把前端占满了，占用率奇高但是对后台的持续计算没什么太大影响

**Reply by M1saka10010** [reply]
我以前的26x台式机也是，我想补到144但是不行，但是cpu占用其实不到50的

**Comment by jiang-yuan** [comment]
我9750h风扇也起飞，但是没4600u这样占用100%啊
我是补60而已，没到100那么高

**Reply by M1saka10010** [reply]
我的轻薄本是4600u的马甲5500u，并不会100%，也就起个飞，感觉会不会是场外因素？

**Reply by jiang-yuan** [reply]
你是4k视频吗？
我试那个14g的电影

**Reply by M1saka10010** [reply]
哦4k啊，那没事了。。。。。。。要不给个链接，我下来试试？

**Reply by M1saka10010** [reply]
以及amd核显的解码能力不太行你是知道的吧。。也有可能是核显解码的锅变成混合解码了

**Reply by jiang-yuan** [reply]
> 以及amd核显的解码能力不太行你是知道的吧。。也有可能是核显解码的锅变成混合解码了

https://btbtt12.com/thread-index-fid-1183-tid-4459527.htm

这个视频,比较老了,可能需要离线下载才能下得动

**Reply by natural-harmonia-gropius** [reply]
我怀疑内存性能瓶颈了，cpu占用的时间都在等待，不用硬解copy用软解试试

**Reply by jiang-yuan** [reply]
小新pro13,三星的内存,比我9750h那台神舟的还好,不至于吧?

**Reply by M1saka10010** [reply]
> > 以及amd核显的解码能力不太行你是知道的吧。。也有可能是核显解码的锅变成混合解码了
> 
> https://btbtt12.com/thread-index-fid-1183-tid-4459527.htm
> 
> 这个视频,比较老了,可能需要离线下载才能下得动

emm，我莫得离线下载，不用各大网盘TTATT

**Reply by jiang-yuan** [reply]
不用网盘的话只能挂下载机才能下得动4k啊,不然好费劲

**Reply by M1saka10010** [reply]
> 小新pro13,三星的内存,比我9750h那台神舟的还好,不至于吧?

核显是用内存的，独显自己有显存。gddr6可比d4快多了

**Reply by M1saka10010** [reply]
> 不用网盘的话只能挂下载机才能下得动4k啊,不然好费劲
在挂了在挂了，不知道啥时候才能下下来，现在速度很慢

**Reply by jiang-yuan** [reply]
你有试过补十几G的4k吗?怎么样

**Reply by M1saka10010** [reply]
正好手上有个鬼灭的，等下下课试试，上课不敢开电脑

**Reply by natural-harmonia-gropius** [reply]
> 小新pro13,三星的内存,比我9750h那台神舟的还好,不至于吧?

建议搞个干净的win10系统，bios关掉tpm和虚拟化再试试

**Reply by jiang-yuan** [reply]
> 

要用模拟器没法关虚拟化
系统是刚重装完的

**Reply by M1saka10010** [reply]
> 你有试过补十几G的4k吗?怎么样

感觉还行？<a href="https://imgtu.com/i/oGrqG8"><img src="https://z3.ax1x.com/2021/12/01/oGrqG8.md.png" alt="oGrqG8.md.png" border="0"></a>

**Reply by jiang-yuan** [reply]
你用的几号补帧方案?我是3.9GHz跑满了,等你下完那个文件再试试看吧

**Reply by M1saka10010** [reply]
> 你用的几号补帧方案?我是3.9GHz跑满了,等你下完那个文件再试试看吧

5
